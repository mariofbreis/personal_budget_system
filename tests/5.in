i 60 0 10 r Receita0
i 60 1 10 r Receita1
i 60 2 10 r Receita2
i 60 3 10 r Receita3
i 60 4 10 r Receita4
i 60 5 10 r Receita5
i 60 6 10 r Receita6
i 60 7 10 r Receita7
i 60 8 10 r Receita8
i 60 9 10 r Receita9
i 60 10 10 r Receita10
i 60 11 10 r Receita11
i 60 12 10 r Receita12
i 60 13 10 r Receita13
i 60 14 10 r Receita14
i 60 15 10 r Receita15
i 60 16 10 r Receita16
i 60 17 10 r Receita17
i 60 18 10 r Receita18
i 60 19 10 r Receita19
i 60 20 10 r Receita20
b 0
b 1
b 2
b 3
b 4
b 5
b 6
b 7
b 8
b 9
b 10
b 11
b 12
b 13
b 14
b 15
b 16
b 17
b 18
b 19
b 20
b 21
b 22
b 23
b 24
b 25
b 26
b 27
b 28
b 29
b 30
b 31
b 32
b 33
b 34
b 35
b 36
b 37
b 38
b 39
b 40
b 41
b 42
b 43
b 44
b 45
b 46
b 47
b 48
b 49
b 50
b 51
b 52
b 53
b 54
b 55
b 56
b 57
b 58
b 59
b 60
b 61
b 62
b 63
b 64
b 65
b 66
b 67
b 68
b 69
b 70
b 71
b 72
b 73
b 74
b 75
b 76
b 77
b 78
b 79
b 80
b 81
b 82
b 83
b 84
b 85
b 86
b 87
b 88
b 89
b 90
b 91
b 92
b 93
b 94
b 95
b 96
b 97
b 98
b 99
b 100
