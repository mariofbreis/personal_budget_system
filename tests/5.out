60
120
180
240
300
360
420
480
540
600
720
840
960
1080
1200
1320
1440
1560
1680
1800
1980
2100
2220
2340
2460
2580
2700
2820
2940
3060
3240
3360
3480
3600
3720
3840
3960
4080
4200
4320
4500
4620
4740
4860
4980
5100
5220
5340
5460
5580
5760
5880
6000
6120
6240
6360
6480
6600
6720
6840
7020
7140
7260
7380
7500
7620
7740
7860
7980
8100
8280
8400
8520
8640
8760
8880
9000
9120
9240
9360
9540
9660
9780
9900
10020
10140
10260
10380
10500
10620
10800
10920
11040
11160
11280
11400
11520
11640
11760
11880
12060
